locales/en/exposure-scan.ftl

```diff
@@ -0,0 +1,34 @@
+exposure-landing-hero-heading = Find out if your personal information has been compromised
+exposure-landing-hero-lead = Stay safe with privacy tools from the makers of { -brand-firefox } that protect you from hackers and companies that publish and sell your personal information. We’ll alert you of any known data breaches, find and remove your exposed info and continually watch for new exposures.
+exposure-landing-hero-email-label = Email address
+exposure-landing-hero-email-placeholder = Enter email address
+exposure-landing-hero-cta-label = Check for breaches
+
+exposure-landing-result-loading = Loading, please wait…
+
+# Variables:
+#   $email (string) - The user's email address, used to identify their data in breaches
+#   $count (number) - Number of data breaches in which the user's data was found
+exposure-landing-result-hero-heading =
+    { $count ->
+        [one] We found <email>{ $email }</email> exposed in <count>1</count> data breach.
+       *[other] We found <email>{ $email }</email> exposed in <count>{ $count }</count> data breaches.
+    }
+
+exposure-landing-result-card-added = Breach Added:
+exposure-landing-result-card-data = Exposed Data:
+exposure-landing-result-card-nothing = No breaches found
+
+exposure-landing-result-footer-attribution = Breach data provided by <hibp-link>{ -brand-HIBP }</hibp-link>
+
+exposure-landing-result-overflow-hero-lead = Sign in to get clear steps on how to resolve these breaches, view all breaches, and get continuous monitoring for any new known breaches.
+exposure-landing-result-overflow-hero-cta-label = Sign in to resolve breaches
+exposure-landing-result-overflow-footer-cta-label = Sign in to view all
+
+exposure-landing-result-some-hero-lead = Sign in to get clear steps on how to resolve these breaches, view all breaches, and get continuous monitoring for any new known breaches.
+exposure-landing-result-some-hero-cta-label = Sign in to resolve breaches
+exposure-landing-result-some-footer-cta-label = Sign up for alerts
```

Comment:
I only see this as a button when there are zero alerts. Where is the hero text in the two strings above displayed? If the context is exactly the same as the previous group, we shouldn't duplicate the strings.

Reply:
Sorry, I messed these up while juggling the six different CTAs: bab6a2f90269db32e430599808c7701ea2f72fb3

The strings above (`hero-lead` and `hero-cta-label`) are these:

![image](https://user-images.githubusercontent.com/4251/230358397-fb1730aa-18a9-417d-80c1-431a4faf29dd.png)

The lead text is the same for >6 and <=6 detected breaches, but different from 0 detected breaches. The other strings differ for all three cases, as far as I'm aware, except for the header, which is the same for each (and uses the same string).

So... Not sure, when does something count as the same context? I can deduplicate `exposure-landing-result-overflow-hero-lead` and `exposure-landing-result-some-hero-lead`, but not `exposure-landing-result-none-hero-lead`, even though they're all used in the same position on the page.